examples/with-style-sheet/babel.config.js

```diff
@@ -0,0 +1,23 @@
+const path = require('path')
+
+module.exports = {
+  presets: [
+    [
+      'next/babel',
+      {
+        'preset-react': {
+          pragma: 'createElement',
+        },
```

Comment:
Is this necessary?

Reply:
yes it is necessary to use the `css` prop